### ISSUE TITLE
Library isn't working with latest phantomjs 2.5.0-beta

### ISSUE THREAD

**Issue by rosolko** [bonigarcia/webdrivermanager#96]
PhantomJS [release](https://groups.google.com/forum/m/#!topic/phantomjs/AefOuwkgBh0) new beta version.

And library isn't work with it. Throw exception:
```java
06:33:09.504 [main] INFO  io.github.bonigarcia.wdm.BrowserManager - Latest version of [phantomjs] is 2.5.0
06:33:09.583 [main] INFO  io.github.bonigarcia.wdm.Downloader - Downloading https://npm.taobao.org/mirrors/phantomjs/phantomjs-2.5.0-beta-linux-ubuntu-xenial-x86_64.tar.gz to /var/jenkins_home/.m2/repository/webdriver/phantomjs/beta-linux-ubuntu-xenial-x86_64/2.5.0/phantomjs-2.5.0-beta-linux-ubuntu-xenial-x86_64.tar.gz
Exception in thread "main" java.lang.RuntimeException: java.lang.IllegalArgumentException: /var/jenkins_home/.m2/repository/webdriver/phantomjs/beta-linux-ubuntu-xenial-x86_64/2.5.0/phantomjs-2.5.0-beta-linux-ubuntu-xenial-x86_64.tar.gz is not a directory
	at io.github.bonigarcia.wdm.BrowserManager.manage(BrowserManager.java:284)
	at io.github.bonigarcia.wdm.BrowserManager.setup(BrowserManager.java:108)
	at io.github.bonigarcia.wdm.BrowserManager.setup(BrowserManager.java:87)
	at com.rosolko.lift.Main.getDriver(Main.java:92)
	at com.rosolko.lift.Main.main(Main.java:31)
Caused by: java.lang.IllegalArgumentException: /var/jenkins_home/.m2/repository/webdriver/phantomjs/beta-linux-ubuntu-xenial-x86_64/2.5.0/phantomjs-2.5.0-beta-linux-ubuntu-xenial-x86_64.tar.gz is not a directory
	at org.apache.commons.io.FileUtils.verifiedListFiles(FileUtils.java:1704)
	at org.apache.commons.io.FileUtils.cleanDirectory(FileUtils.java:1674)
	at org.apache.commons.io.FileUtils.deleteDirectory(FileUtils.java:1575)
	at io.github.bonigarcia.wdm.Downloader.checkPhantom(Downloader.java:319)
	at io.github.bonigarcia.wdm.Downloader.extract(Downloader.java:236)
	at io.github.bonigarcia.wdm.Downloader.download(Downloader.java:131)
	at io.github.bonigarcia.wdm.BrowserManager.manage(BrowserManager.java:277)
	... 4 more
:run FAILED
```

**Comment by elekdavid** [comment]
Same here

**Comment by FranAguiar** [comment]
Fix: PhantomJsDriverManager.getInstance().setup("phantomjs-2.1.1");

**Comment by elekdavid** [comment]
Thanks, it solved the problem for me.

**Comment by rosolko** [comment]
But it isn't solve problem. Just hardly set older version of phantom as working.

**Comment by FranAguiar** [comment]
Yes, I mean as a hotfix while isn't working properly

If you don't need phantom 2.5 beta it works

**Comment by bonigarcia** [comment]
The packaging name convention for PhantomJS 2.5.0-beta has changed, breaking the compatibility with WebDriverManager 1.5.0. Moreover it seems to be two different phantomjs binaries for Ubuntu (Trusty and Xenial), making things a bit harder.

I have to code commit new patches to support it, but so far this fix (equivalent to the proposed by @FranAguiar) should to do the trick:

`PhantomJsDriverManager.getInstance().setup("2.1.1");`

**Comment by bonigarcia** [comment]
I've just released version 1.5.1. This should be fixed now. Nevertheless, I checked that PhantomJS is working fine in Windows but it is not on Ubuntu (at least in Trusty distro). More info [here](http://stackoverflow.com/questions/41675982/phantomjs-2-5-0-beta-for-selenium-webdriver-not-working-in-linux).
